src/Features/Core/Portable/AddAccessibilityModifiers/AbstractAddAccessibilityModifiersCodeFixProvider.cs

```diff
@@ -51,17 +51,10 @@ public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
                 editor.ReplaceNode(
                     declaration,
                     (currentDeclaration, generator) =>
-                    {                        
-                        if(generator.GetAccessibility(currentDeclaration) == Accessibility.NotApplicable)
-                        {
-                            // No accessibilty was declared, we need to add it
-                            return generator.WithAccessibility(currentDeclaration, symbol.DeclaredAccessibility);
-                        }
-                        else
-                        {
-                            // There was an accessibility, so remove it
-                            return generator.WithAccessibility(currentDeclaration, Accessibility.NotApplicable);
-                        }                        
+                    {
+                        return generator.GetAccessibility(currentDeclaration) == Accessibility.NotApplicable ?
```

Comment:
nit, Roslyn pattern is to place ? and : on the next lines, aligned with each other :)

Reply:
If only there were rules for this :)

Reply:
Fixed

Reply:
> If only there were rules for this :)

'twoudbenice!